igvm/vm.py

```diff
@@ -672,54 +740,22 @@ def aws_build(self,
         disk_size_gib = self.dataset_obj['disk_size_gib']
 
         for vm_type in vm_types:
```

Comment:
What about just appending `c5.xlarge` to the `vm_types` list? Would be less code with the same result. I would also load the vm_type from the settings and make this a variable, as there's already `c6i` (Intel) and `c6a` (AMD) available, so you have a central place to replace the failback vm type.